docs/components/board/_index.md

```diff
@@ -420,7 +420,7 @@ err := myBoard.Status(context.Background(), nil)
 {{% /tab %}}
 {{< /tabs >}}
 
-### SetPWM
+### SetPowerMode
```

Comment:
We go by the names defined in the proto: https://github.com/viamrobotics/api/blob/main/component/board/v1/board_grpc.pb.go#L33

are those changing?

Reply:
The text under this is not decribing the correct proto method - it's describing https://github.com/viamrobotics/api/blob/80ccf2fe7a14d8bcdc3075b9e2b81f2458e6928f/component/board/v1/board_grpc.pb.go#L49

Reply:
the actual `SetPWM` API is documented in line 880 of this file

Reply:
ahh - great catch - thanks!